### ISSUE TITLE
File reopen with file moved/removed

### ISSUE THREAD

**Issue by kenchong48** [rrivirr/rriv#113]
Symptom: file with no header (happened on 5 out of 6 boards in a floating bin experiment)

Potential causes:
Files moved/removed manually while still deployed
That in combination with a manual reset or maybe lack of manual reset, since a manual reset should result in a new file being generated?

Does the file system successfully reopen a file even if the file does not exist, generating it in the process?

I was copying data off the SD cards then moving them to Old Data folders.  I was doing this in between changing lab conditions of the floating bucket, so did not see a need to redeploy them.  This would be an issue for field deployments that follow the same idea of not redeploying each time.

**Comment by ZavenArra** [comment]
Makes sense.  I don't think we've consider this use case where files are moved but deployment isn't halted.
